### PR TITLE
[GH-8] Added body to the details of http context

### PR DESCRIPTION
**Pull request by annetteres**
Implements https://github.com/soundcloud/periskop/issues/8

### CODE REVIEW

**Comment by marctc** [comment]
We need to update also the [proto definition](https://github.com/soundcloud/periskop/blob/master/representation/errors.proto).

**Review by rciovati — APPROVED** [review]
Looks good to me! 👍👍 👍 

(@marctc the proto is already updated 🤔 )

File: representation/errors.proto [hunk 1]
```diff
@@ -31,4 +31,5 @@ message HttpContext {
     string requestMethod = 1;
     string requestUrl = 2;
     map<string, string> requestHeaders = 3;
+    string requestBody = 4; // Optional
```
Comment (annetteres) [inline representation/errors.proto:34]: @marctc here, no?

**Comment by marctc** [comment]
> Looks good to me! +1+1 +1
> 
> (@marctc the proto is already updated thinking )

GIF

**Review by marctc — APPROVED** [review]
Nice :+1:

**Pull request merged**